pyroSAR/spatial/get_tile_from_url.py

```diff
@@ -0,0 +1,199 @@
+"""
```

Comment:
Please exclude this module from the pull request. This is a different work package and still ongoing work

Reply:
Done.